proposals/lip-update_genesis_block_schema_and_processing.md

```diff
@@ -0,0 +1,169 @@
+
+## Rationale
+
+### Usage of `assets` property of the block 
+
+The new genesis block format and processing is specified with the rationale of having a compact and self-contained way of initializing a blockchain in the Lisk ecosystem. 
+With this in mind, this LIP defines a genesis block format and processing based on the specifications given in [Update block schema and block processing LIP][BlockLIP]. 
+In particular, the `assets` property contains the necessary information to initialize the state of the blockchain. 
+Each element in the `assets` array contains the information necessary to set the state store for a given module. 
+Hence, each module should define the format and processing logic for this information in the genesis block.
+
+### Processing of the genesis block 
+
+Another distinctive specification of the genesis block is its processing. 
+The blockchain starts with an empty key-value store and in particular, all module stores are empty initially. 
+As part of the genesis block processing modules can add key-value entries to their module store to initialize their state. 
+They should also check the consistency of their state independently and against information provided by other modules. 
+Also, before this is done, the framework layer has to validate the block header and the block format itself. 
+Hence, this LIP defines steps specific to the processing of the genesis block: a step to initialize the state store per each module and the second step to verify this initial state. 
+In particular the processing of the genesis block happens in four steps described in the sections below.
+
+<img src="lip-update_genesis_block_schema_and_processing/stages.png" width="70%">
+
+*Figure 1: A schematic depiction of the steps of the processing of the genesis block.*
+
+#### Static validation of the genesis block
+
+The genesis block has to go through initial static checks to ensure that the serialized object follows the general structure of a block. 
+Also, certain properties of the block header are checked at this stage. 
+All of these checks are stateless since the state of the blockchain is yet to be initialized. 
+The key differences as compared to [the validation for the rest of the blocks][BlockLIPvalidation] in a blockchain are that there is no specific size limit for the genesis block object and that the genesis block should not contain any transaction.
+
+#### Genesis state Initialization
+
+At this stage, the genesis state initialization logic for all registered modules is executed. 
+For this purpose, registered modules can specify processing logic considering the information in their corresponding entry of the `assets` property. 
+Typically, modules will perform format and data consistency checks of their respective element in `assets` and then initialize their state according to the data provided in it. 
```

Comment:
To be consistent with the term `entry` above and `entries` in previous paragraph
```suggestion
Typically, modules will perform format and data consistency checks on their corresponding entry in `assets` and then initialize their state according to the data provided in it. 
```

Reply:
fixed